packages/variants/src/variants.ts

```diff
@@ -15,6 +16,13 @@ function makeThemeFromPartials(
   });
 }
 
+function changeOpacity(inputColor: string, opacity: number): string {
+  const newColor: Colors.IColor | undefined = Colors.getColorFromString(inputColor);
+  newColor!.a = opacity;
+  const outputRBGAString = `rgb(${newColor!.r},${newColor!.g},${newColor!.b},${newColor!.a})`;
```

Comment:
you can just say newColor.string, it should autopopulate with an rgba for you

Reply:
@phkuo `const outputRBGAString = newColor.string;` gives me the ts error "Property 'string' does not exist on type 'IColor'." Can you clarify your suggestion?

Reply:
sorry i meant newColor.str
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/utilities/color/colors.ts#L30

Reply:
@bennettclark also you should say newColor = updateA(newColor, opacity) so it will actually populate .str for you:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/utilities/color/colors.ts#L241

Reply:
actually thing entire function can be replaced with the utilities there, see other comment